file_paths.md

```diff
@@ -280,6 +280,13 @@ These sections describe other Puppet packages that rely on puppet-agent to creat
                 server_datadir            # :server_data
                 yaml                      # :yamldir
 
+    /var/log/puppetlabs                   # :logdir                      /var/lib/puppet/log
+        puppetserver/                     # writeable by puppetserver
+            puppetserver.log
```

Comment:
puppetdb runs as the puppetdb user, so it would have the same problem. Making /var/log/puppetlabs owned by root:puppet isn't feasible because the puppet-agent package doesn't make the puppet user/group.